next.config.mjs

```diff
@@ -22,28 +22,20 @@ const nextConfig = {
     images: {
       unoptimized: true,
     },
-    modularizeImports: {
-      '@mui/material': {
-        transform: '@mui/material/{{member}}',
-      },
-      '@mui/icons-material/?(((\\w*)?/?)*)': {
-        transform: '@mui/icons-material/{{ matches.[1] }}/{{member}}',
-      },
-      lodash: {
-        transform: 'lodash/{{member}}',
-      },
-      'date-fns': {
-        transform: 'date-fns/{{member}}',
-      },
-    },
   },
-  async rewrites() {
-    return [
-      {
-        source: '/:safe([a-z0-9-]+\\:0x[a-fA-F0-9]{40})/:path*',
-        destination: '/:path*?safe=:safe',
-      },
-    ]
+  modularizeImports: {
+    '@mui/material': {
+      transform: '@mui/material/{{member}}',
+    },
+    '@mui/icons-material/?(((\\w*)?/?)*)': {
+      transform: '@mui/icons-material/{{ matches.[1] }}/{{member}}',
+    },
+    lodash: {
+      transform: 'lodash/{{member}}',
+    },
+    'date-fns': {
+      transform: 'date-fns/{{member}}',
+    },
```

Comment:
Moving this out of the experimental property somehow makes it load a lot more modules again (17k instead of 6k) and slowing down local development. Do we also have to upgrade nextjs for it to work?

Reply:
I'm seeing an increase in chunk sizes due to the upgraded packages (from 900 KB to 1.01 MB) regardless of this change.

Reply:
Without this, the build says:
> warn  - You have defined experimental feature (modularizeImports) in next.config.mjs that does not exist in this version of Next.js.

Reply:
Anyway, let's not touch it for now then.